packages/block-library/src/query/index.php

```diff
@@ -31,8 +31,17 @@ function render_block_core_query( $attributes, $content, $block ) {
 			// Add the necessary directives.
 			$p->set_attribute( 'data-wp-interactive', 'core/query' );
 			$p->set_attribute( 'data-wp-router-region', 'query-' . $attributes['queryId'] );
-			$p->set_attribute( 'data-wp-context', '{}' );
+			$p->set_attribute(
+				'data-wp-context',
+				wp_json_encode(
+					[
+						'routerRegion' => 'query-' . $attributes['queryId'],
+						'isLoading'    => false,
```

Comment:
I think we have to find a better name, since all the regions are going to navigate/load. It's just that this is the region that has started the navigation.

Is there a way to make that more obvious?